app/Database/Seeds/SampleDataSeeder.php

```diff
@@ -13,42 +13,42 @@
-        'General Help'                       => ['parent' => 'Using CodeIgniter', 'order' => 6, 'active' => 1, 'private' => 0],
-        'CodeIgniter 4'                      => ['parent' => null, 'order' => 4, 'active' => 1, 'private' => 0],
-        'CodeIgniter 4 Roadmap'              => ['parent' => 'CodeIgniter 4', 'order' => 1, 'active' => 1, 'private' => 0],
-        'CodeIgniter 4 Development'          => ['parent' => 'CodeIgniter 4', 'order' => 2, 'active' => 1, 'private' => 0],
-        'CodeIgniter 4 Feature Requests'     => ['parent' => 'CodeIgniter 4', 'order' => 3, 'active' => 1, 'private' => 0],
-        'CodeIgniter 4 Discussion'           => ['parent' => 'CodeIgniter 4', 'order' => 4, 'active' => 1, 'private' => 0],
-        'CodeIgniter 4 Addins'               => ['parent' => 'CodeIgniter 4', 'order' => 5, 'active' => 1, 'private' => 0],
-        'Development'                        => ['parent' => null, 'order' => 5, 'active' => 1, 'private' => 0],
-        'CodeIgniter 3.x'                    => ['parent' => 'Development', 'order' => 1, 'active' => 1, 'private' => 0],
-        'CodeIgniter 2.x'                    => ['parent' => 'Development', 'order' => 2, 'active' => 1, 'private' => 0],
-        'Issues'                             => ['parent' => 'Development', 'order' => 3, 'active' => 1, 'private' => 0],
-        'Netbeans Plugin'                    => ['parent' => 'Development', 'order' => 4, 'active' => 1, 'private' => 0],
-        'External Resources'                 => ['parent' => null, 'order' => 6, 'active' => 1, 'private' => 0],
-        'Spotlight'                          => ['parent' => 'External Resources', 'order' => 1, 'active' => 1, 'private' => 0],
-        'Learn More'                         => ['parent' => 'External Resources', 'order' => 2, 'active' => 1, 'private' => 0],
-        'Jobs'                               => ['parent' => 'External Resources', 'order' => 3, 'active' => 1, 'private' => 0],
-        'Addins'                             => ['parent' => 'External Resources', 'order' => 4, 'active' => 1, 'private' => 0],
-        'Archived Discussions'               => ['parent' => null, 'order' => 7, 'active' => 1, 'private' => 0],
-        'Archived General Discussion'        => ['parent' => 'Archived Discussions', 'order' => 1, 'active' => 1, 'private' => 0],
-        'Archived Libraries & Helpers'       => ['parent' => 'Archived Discussions', 'order' => 2, 'active' => 1, 'private' => 0],
-        'Archived Development & Programming' => ['parent' => 'Archived Discussions', 'order' => 3, 'active' => 1, 'private' => 0],
+        'Operations'                         => ['parent' => null, 'order' => 1, 'active' => 1, 'permissions' => ['admin.access', 'categories.moderate']],
+        'Framework Planning'                 => ['parent' => 'Operations', 'order' => 1, 'active' => 1, 'permissions' => null],
```

Comment:
If we're mimicing the current forums, then the next 3 categories would also be restricted.

Reply:
If a child category does not have permissions, it will inherit them from the parent category. I thought this would make sense to work that way.